redaxo/src/core/lib/form/elements/element.php

```diff
@@ -311,6 +314,19 @@ public function getValidator()
         return $this->validator;
     }
 
+    /**
+     * @return bool
+     */
+    public function getLabelOnTop()
```

Comment:
Ich würde hier den `is`-Präfix nutzen, und nativen Returntype:

```suggestion
    public function isLabelOnTop(): bool
```

Selbst wenn es nur die zwei Varianten gibt, könnte man es auch trotzdem so machen, wie Thomas vorschlägt, mit Konstanten. Wobei `Alignment` für mich auch nicht so klar ist. Könnte sich auch auf die Textausrichtung im Feld beziehen.

Daher kann es wegen mir auch bei `labelOnTop` bleiben.